app/presenters/fields_of_operation_presenter.rb

```diff
@@ -0,0 +1,14 @@
+class FieldsOfOperationPresenter < ContentItemPresenter
+  include ContentItem::TitleAndContext
+
+  def fields_of_operation
+    content_item.dig("links", "fields_of_operation").map { |field| [field["title"], field["base_path"]] }
+  end
+
+  def title_and_context
+    super.tap do |t|
+      t[:context] = I18n.t("fields_of_operation.context")
+      t.delete(:average_title_length)
```

Comment:
I know this is in a few places, what's it actually doing?

Reply:
It affects the size of the title: see [here](https://github.com/alphagov/govuk_publishing_components/blob/b2ffbd3d448579f26f0802436a467e59fe40b301/app/views/govuk_publishing_components/components/_title.html.erb#L22). Given we know the title here is always of a fixed length, and short, I removed this element.